admin/class-automation.php

```diff
@@ -106,6 +96,8 @@ public static function action__admin_menu(): void {
 			self::PAGE_NAME,
 			[ __CLASS__, 'render_submenu_page' ]
 		);
+
+		add_action( "admin_print_scripts", [ __CLASS__, 'add_options_assets' ] );
```

Comment:
This action should be registered in the `init` function and should follow the same format, where the callback is `action__` followed by the action name.